src/pages/resources/reference/push-notifications/templates/tabs/zero_bezel.md

```diff
@@ -0,0 +1,91 @@
+| Priority | ⛔️ | `adb_n_priority` | string | Designates the notification priority for Android - [importance](https://developer.android.com/reference/android/app/NotificationChannel#setImportance(int)) for API >= 26 (Android 8.0) or [priority](https://developer.android.com/reference/androidx/core/app/NotificationCompat.Builder#setPriority(int)) for API < 26 |
+| Tag | ⛔️ | `adb_tag` | string | Identifier used to replace existing notifications in the notification drawer. If not specified, each request creates a new notification. If specified and a notification with the same tag is already being shown, the new notification replaces the existing one in the notification drawer. |
+| Sticky | ⛔️ | `adb_sticky` | string | Boolean value represented as a string "true" or "false".<br />When set to false or unset, the notification is automatically dismissed when the user clicks it in the panel. When set to true, the notification persists even when the user clicks it. |
+| Ticker | ⛔️ | `adb_ticker` | string | Sets the "ticker" text, which is sent to accessibility services. Prior to API level 21 (Lollipop), sets the text that is displayed in the status bar when the notification first arrives. |
+| Color - Small Icon | ⛔️ | `adb_clr_icon` | string | Color for the notification's small icon.<br />Represented as six character hex, e.g. `00FF00` |
+| Color - Background | ⛔️ | `adb_clr_bg` | string | Color for notification's background.<br />Represented as six character hex, e.g. `00FF00` |
+| Collapsed notification style | ⛔️ | `adb_col_style` | string | Determines whether the collapsed message should show a zero-bezel image, or text only. |
+
+## Example
+
+<img src="./../assets/zero_bezel.gif" width="50%" height="50%"/>
+
+Below is a sample of what a payload might look like for a zero bezel notification:
+
+```json
+{
+  "message": {
+    "token": "FCM_TOKEN",
+    "android": {
+      "collapse_key": "new message"
+    },
+    "data": {     
+      "adb_version": "1",
+      "adb_title": "game request",
+      "adb_body": "shall we play a game?",               
+      "adb_sound": "bingBong",
+      "adb_icon": "ic_knight",
+      "adb_n_count": "1",
+      "adb_n_priority": "PRIORITY_LOW",
+      "adb_image": "https://pictureofchess.com/board.png",
+      "adb_uri": "https://chess.com/games",
+      "adb_a_type": "WEBURL",
+      "adb_template_type": "zb",
+      "adb_body_ex": "Bob wants to play a game of chess with you. Click the notification to accept!",
+      "adb_clr_body": "00EE00",
+      "adb_clr_title": "AABBCC",
+      "adb_clr_icon": "123456",
+      "adb_col_style": "img"
+    }
+  }
+}
+```
+
+## Usage recommendations
+
+The tables below contain guidelines for your push notification content. **These recommendations help your text and images display reliably across multiple devices.** These values are guidelines only - it is still recommended to test a notification prior to sending it.
```

Comment:
Similar comment for the it is recommended as above: "... guidelines only - you should still test a notification..."

Reply:
cleaned this is all the docs within the `tabs` directory